libpius/path.py

```diff
@@ -76,13 +76,13 @@ def _access_check(fn, mode):
 #end of "Which" function
 
 def gpg_path():
-    if sys.platform = "win32":
+    if sys.platform == "win32":
         gpg = which('gpg2')
-    elif sys.platform = "darwin":
+    elif sys.platform == "darwin":
         #joining all possible paths for location of mac binaries
-        BIN_PATHS = os.path.join(MAC_BIN_PATHS, LINUX_BIN_PATHS)
+        BIN_PATHS = MAC_BIN_PATHS+";"+LINUX_BIN_PATHS)
```

Comment:
Since most people using gpg and hacking on this code base are familiar with linux over windows, lets use `:` as the PATH separator?

Reply:
They both resolve the same way as python is able to handle it both ways, possibly more, as long as it knows what the delimiter is.